pilot/pkg/features/pilot.go

```diff
@@ -704,6 +704,10 @@ var (
 
 	StackdriverAuditLog = env.Register("STACKDRIVER_AUDIT_LOG", false, ""+
 		"If enabled, StackDriver audit logging will be enabled.").Get()
+
+	PersistOldestWinsHeuristicForTrafficPolicy = env.Register("PERSIST_OLDEST_FIRST_HEURISTIC_FOR_TRAFFIC_POLICY", false,
```

Comment:
PERSIST_OLDEST_FIRST_HEURISTIC_FOR_TRAFFIC_POLICY seems very generic traffic policy flag. Should we be more specific like 
`PERSIST_OLDEST_FIRST_HEURISTIC_FOR_WILD_CAR_HOST_MATCHING` ?

Reply:
Oldest wins is a common heuristic throughout the codebase; I thought the flag could be more generally useful if we need to migrate away from that in the future, but I don't feel super strongly. I'll err on the side of specificity and take your suggestion

Reply:
> Oldest wins is a common heuristic throughout the codebase; I thought the flag could be more generally useful if we need to migrate away from that in the future,

I doubt we will change any of that in future :-). Being specific is better IMO

Reply:
Agreed. Changed